languages/elixir/exercises/concept/multiple-clause-functions/.meta/design.md

```diff
@@ -0,0 +1,61 @@
+# Concept Exercise: multiple-clause-functions
+
+## Goal
+
+The goal for this exercise is to teach more about functions in elixir and how to use multiple functions clauses, guards
+
+## Learning objectives
+
+A student upon completion will:
+
+- Know what a guard is.
+- Know how to use a guard.
+- Know that guards return strict boolean values.
+- Know that guards meet certain guarantees to not mutate state.
+- Know that guards must be declared in a special form, and not all functions that return boolean values are guards.
+- Know how to write basic multiple-clause functions.
+- Know how to use guards to protect execution of a function.
+- Know how to use guards to perform basic control flow.
+- Know how to specify a default argument
+- Know that by specifying a default argument, it defines multiple arities for the function name
+
+## Out of scope
+
+- pattern matching
+
+## Concepts
+
+- `atoms`
+- `multiple-clause-functions`
+- `guards`
+- `default-parameters`
```

Comment:
The `maps` exercise has `default-arguments` listed as a prerequisite, I would stick with that name as it's also used in the official guides.

Reply:
I changed it everywhere else, missed this one, thanks!